crates/pathfinder/src/consensus/inner.rs

```diff
@@ -24,10 +24,10 @@ pub fn start(
 ) -> ConsensusTaskHandles {
     // Events that are produced by the P2P task and consumed by the consensus task.
     // TODO determine sufficient buffer size. 1 is not enough.
-    let (tx_to_consensus, rx_from_p2p) = mpsc::channel::<ConsensusTaskEvent>(10);
+    let (tx_to_consensus, rx_from_p2p) = mpsc::channel::<ConsensusTaskEvent>(10000);
     // Events that are produced by the consensus task and consumed by the P2P task.
     // TODO determine sufficient buffer size. 1 is not enough.
-    let (tx_to_p2p, rx_from_consensus) = mpsc::channel::<P2PTaskEvent>(10);
+    let (tx_to_p2p, rx_from_consensus) = mpsc::channel::<P2PTaskEvent>(10000);
```

Comment:
These hardcoded arbitrary channel sizes still feel like will be biting us sooner or later

Reply:
Ah good catch, the buffer size of 10 was sufficient (but the TODO still holds, ofc), I was trying out something else and forgot to revert to 10!